search/mindbender-search

```diff
@@ -24,6 +24,13 @@
 ##
 # Author: Jaeho Shin <netj@cs.stanford.edu>
 # Created: 2015-07-31
+
+##Remark: the default heap size of Elastic search (1GB) may be insufficient for the application, resulting in error "Data too large, data for [type] would be larger than limit of [622775500/593.9mb]".
+# To modifiy it, run:
+# export ES_HEAP_SIZE=10g
+# before mindbender search gui. Pick a value adapted for your machine but lower than 30.5GB. More info: https://www.elastic.co/guide/en/elasticsearch/guide/current/heap-sizing.html
```

Comment:
Sure, done ! However, the comment doesn't seem to appear when I do mindbender search... (it prints everything until "mindbender search show-elasticsearch-mappings", as previously)